modules/apps/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/model/impl/MBMessageImpl.java

```diff
@@ -121,6 +122,27 @@ public int getAttachmentsFileEntriesCount() throws PortalException {
 		return attachmentsFileEntriesCount;
 	}
 
+	@Override
+	public FileEntry getAttachmentsFileEntryByExternalReferenceCode(
+			long groupId, String externalReferenceCode)
+		throws PortalException {
+
+		FileEntry portletFileEntryByExternalReferenceCode =
```

Comment:
Names must end with the type. Just call this `fileEntry` instead of `portletFileEntryByExternalReferenceCode`.